app/src/screens/Developer.tsx

```diff
@@ -166,6 +171,30 @@ const Settings: React.FC = () => {
     setPreventAutoLock((previousState) => !previousState)
   }
 
+  const getPushNotificationCapable = async () => {
+    if (!agent) return
+    if ((await PushNotificationHelper.isMediatorCapable(agent)) === true) setPushNotificationCapable(true)
+    else setPushNotificationCapable(false)
+  }
+
+  const initializePushNotificationsToggle = async () => {
+    setEnablePushNotifications(await PushNotificationHelper.isEnabled())
+  }
+
+  const toggleDevPushNotificationsSwitch = () => {
+    if (!pushNotificationCapable || !agent) return
+    if (enablePushNotifications) PushNotificationHelper.setDeviceInfo(agent, true)
+    else PushNotificationHelper.setup(agent)
```

Comment:
Gonna stop pointing these out because there's a lot of them but I think generally in Bifold and BC Wallet we tend to prefer bracketed `if/else` statements that span lines. Makes it easier to read if it's consistent.

Reply:
Thanks for the review. No problem. Will update the syntax.